analytics/src/docker/docker-entrypoint.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+DIR=/docker-entrypoint.d
+
+# Executing custom scripts located in CUSTOM_SCRIPTS_DIRECTORY if environment variable is set
+if [[ -z "${CUSTOM_SCRIPTS_DIRECTORY}" ]]; then
+  echo "[INFO] No CUSTOM_SCRIPTS_DIRECTORY env variable set"
+else
+  echo "[INFO] CUSTOM_SCRIPTS_DIRECTORY env variable set to ${CUSTOM_SCRIPTS_DIRECTORY}"
+  # Regex is needed in jetty9 images, but not alpine's ones.
```

Comment:
This comment is not necessary anymore I think? And it should maybe moved to the 18th line?